src/DssDirectDepositHub.sol

```diff
@@ -378,16 +371,22 @@ contract DssDirectDepositHub {
             uint256 debt = vat.debt();
             uint256 toUnwind;
             if (Art > lineWad) {
-                toUnwind = Art - lineWad;
+                unchecked {
+                    toUnwind = Art - lineWad;
+                }
             }
             if (debt > Line) {
-                toUnwind = _max(toUnwind, _divup(debt - Line, RAY));
+                unchecked {
```

Comment:
Was wondering if the `unchecked` might modify the behaviour of `_divup`, but no, nothing to worry about:
"The setting only affects the statements that are syntactically inside the block. Functions called from within an unchecked block do not inherit the property."
https://docs.soliditylang.org/en/v0.8.14/control-structures.html